### DISCUSSION TITLE
Where does OpenRadioss store Radioss object?

### DISCUSSION THREAD

**Discussion by halfwitbm** [OpenRadioss/OpenRadioss#1754] [Q&A]
Hi All,

As my understanding on OpenRadioss codes, it transfers variables information by *.inc file, *mod.F file and some temporary scratch files, for LS-DYNA input, OpenRadioss convert it to compatible Radioss object then read information. No matter for Radioss input or for LS-DYNA input, they are all stored in Radioss object. Where does OpenRadioss store these Radioss object? In the memory or in the scratch files? 
I trace the codes and find it locate card title and then read its content, but I didn't find the location information where it read these cards from. As my guess, the input Dect information (No matter Radioss input or LS-DYNA input) will be dumped into memory (omit the comment part) then start reading.  Can someone share me your understanding on this part? Thank you.

Bin

**Comment by elequiniou** [comment]
Hi Bin,

This is very a good point. Thanks a lot for spending time to study this part of the code and for your accurate analysis. I confirm that OpenRadioss stores those objects in memory. We are working on documenting an API to access and modify those objects. We will come back to you as soon as possible with more details. Thanks a lot for you patience and interest in this project!

Eric.

**Comment by halfwitbm** [comment]
Hi Eric,

Thank you very much for the information. It could help me figure out more
deep information on the current source code.

Regards,

Bin
